sdk/sdk-view/src/test/java/mil/emp3/api/PointTest.java

```diff
@@ -0,0 +1,134 @@
+package mil.emp3.api;
+
+import android.test.mock.MockContext;
+import android.util.Log;
+import android.webkit.URLUtil;
+
+import org.cmapi.primitives.GeoIconStyle;
+import org.cmapi.primitives.GeoPoint;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+
+import java.io.File;
+
+import static org.junit.Assert.*;
+import static org.mockito.Matchers.any;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+/**
+ * Created by Matt.Miller on 10/16/2017.
+ */
+@PrepareForTest({URLUtil.class})
+public class PointTest extends TestBase {
+    class MyMockContext extends MockContext {
+        @Override
+        public File getDir(String name, int mode) {
+            Log.d(TAG, "Current Dir " + System.getProperty("user.dir"));
+            return new File(System.getProperty("user.dir") + File.separator + name);
+        }
+    }
+
+    public static final String TAG = PointTest.class.getSimpleName();
+    Point p1;
+    GeoPoint gp;
+
+    @org.junit.Before
+    public void setUp() throws Exception {
+        MockContext context = new PointTest.MyMockContext();
+        PowerMockito.mockStatic(URLUtil.class);
+        when(URLUtil.isValidUrl(any(String.class))).thenReturn(true);
+        p1 = new Point();
+        String s = "127.0.0.1";
+        p1 = new Point(s);
+        gp = new GeoPoint();
+        p1 = new Point(gp);
+        p1 = new Point(50,50);
+        Log.i(TAG, p1.toString());
+
+    }
+
+    @org.junit.Test
+    public void badConstructors() throws Exception {
```

Comment:
these bad constructors should be separated out into separate tests.  Also we should talk about how you should check if a test fails properly.  There is the ability to have a test where it expects an exception and if that exception isn't thrown the the test will fail.  You can look at some of the tests I have written for KMZ for an example.  at the top of the class you will see  (expected = IllegalArgumentException.class)  (double check my syntax). I know you will be in on Thursday, lets find time to talk about this after you made some changes.